### PR TITLE
fix and verify easy-install.pth in EasyBuild install, also install vsc-base pkg

### PR DESCRIPTION
**Pull request by boegel**
more changes to easyblock for installing EasyBuild, caused by #559

@wpoely86: please review

### CODE REVIEW

File: easybuild/easyblocks/e/easybuildmeta.py [hunk 1]
```diff
@@ -81,16 +90,33 @@ def install_step(self):
...
         """Custom sanity check for EasyBuild."""
 
         # list of dirs to check, by package
         # boolean indicates whether dir is expected to reside in Python lib/pythonX/site-packages dir
         subdirs_by_pkg = [
-                          ('framework', [('easybuild/framework', True), ('easybuild/tools', True)]),
-                          ('easyblocks', [('easybuild/easyblocks', True)]),
-                          ('easyconfigs', [('easybuild/easyconfigs', False)]),
-                         ]
+            ('framework', [('easybuild/framework', True), ('easybuild/tools', True)]),
```
Comment (wpoely86) [inline easybuild/easyblocks/e/easybuildmeta.py]: add `vsc-base`?

File: easybuild/easyblocks/e/easybuildmeta.py [hunk 2]
```diff
@@ -81,16 +90,33 @@ def install_step(self):
         except OSError, err:
             self.log.error("Failed to install EasyBuild packages: %s" % err)
 
+        env.setvar('PYTHONPATH', orig_pythonpath)
+
+        # check whether easy-install.pth contains correct entries
```
Comment (wpoely86) [inline easybuild/easyblocks/e/easybuildmeta.py]: something for in the sanity check?

**Comment by boegel** [comment]
without this in place, `easy-install.pth` was 'corrupted', in the sense that the `.egg` dirs of the EasyBuild being used to install with were being found, and injected in `easy-install.pth`:

```
$ cat /apps/gent/SL6/westmere/software/EasyBuild/1.16.1/lib/python2.6/site-packages/easy-install.pth 
import sys; sys.__plen = len(sys.path)
/vscmnt/gent_vulpix/_/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.0.0dev-py2.6.egg
/vscmnt/gent_vulpix/_/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/vsc_base-2.0.3-py2.6.egg
./easybuild_easyconfigs-1.16.1.0-py2.6.egg
/vscmnt/gent_vulpix/_/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.0.0dev-py2.6.egg
```

File: easybuild/easyblocks/e/easybuildmeta.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/e/easybuildmeta.py]: ok, true

File: easybuild/easyblocks/e/easybuildmeta.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/e/easybuildmeta.py]: will do

**Comment by boegel** [comment]
I tested this through by rechecking installation of EB v1.16.1, v2.0.0(dev), and retesting the bootstrap (using v2.0.0dev tarballs), all working fine

so, going in, thanks @wpoely86 for the review!

**Pull request merged**